src/modules/accounting/accounting.py

```diff
@@ -40,11 +57,20 @@ def __init__(self, w3: Web3):
         self.lido_validator_state_service = LidoValidatorStateService(self.w3)
         self.bunker_service = BunkerService(self.w3)
 
+    def refresh_contracts(self):
+        self.report_contract = self.w3.lido_contracts.accounting_oracle
+
+    def clear_cache(self):
```

Comment:
Make decorartor with global state with all caches.
Create function that clears all cache in decorator's instance

Reply:
:warning: Clear cache contracts